### ISSUE TITLE
https invite links may not be working again 

### ISSUE THREAD

**Issue by n8fr8** [zom/Zom-iOS#180]
https://zom.im/i/#### invite links seem to not be getting opened again in the Zom app. Then, when they go to the invite page on the site, the xmpp: uri links triggered by the "add friend" button are not working either.

Maybe some issue with general invite link handling?

@N-Pex can you try to repro?

I will do some more testing here, but we have multiple reports coming in on this.

**Comment by N-Pex** [comment]
@n8fr8 I can't repro this, links work fine on my iPad (note: only tested on debug build)

Universal links don't work on iOS 8, could they still be running this?

**Comment by n8fr8** [comment]
No. I think this is related to some apps, like Gmail, opening links directly in Chrome browser. Then, we show the invite web page, and the "add friend" button. For some reason, that button is not triggering the xmpp: uri properly to Zom. At least, that is my current guess. Will get more details.

**Comment by gekhung** [comment]
"when people send me a link to add them on zom on text or facebook msg it takes me to the page with their qr code and user name but when i click on the qr code or the ‘Add me’ under the qr code it opens zom and nothing happens - they don’t get added.

The only way for me to add them then is to memorize their user name and manually type it in which gets cumbersome with the long usernames."

**Comment by n8fr8** [comment]
I think the link is opening in the browser, because Gmail and others do this direclty, and don't respect the universal links. 

Then, on the site, we use the xmpp: links to trigger Zom properly, but for some reason, the users are not being added into Zom iOS.

xmpp:foo@domain.com?subscribe@otr-fingerprint=12345 

is the link that should work.

**Comment by N-Pex** [comment]
@n8fr8 I guess you mean xmpp:foo@domain.com?subscribe&otr-fingerprint=12345? so that query parameters are separated correctly?

**Comment by N-Pex** [comment]
@n8fr8 Looking further into this, what we get in the client from the zom.im page when clicking on the "add friend" link (or QR code) is the URL "xmpp:?subscribe&otr-fingerprint=x.2e4c8e48@dukgo.com%1B%DD%9C%E...."

This seems wrong(?) Current upstream CS code requires it to be xmpp:user@server.com?subscribe... 

Another thing I notice is that I am testing with username npex.2e4c8e48@dukgo.com, yet the otr-fingerprint starts with x.2e4c... Is this correct?

**Comment by n8fr8** [comment]
I mean the Zom app itself handling the https  invite link from iMessage.
I think it is now, but the website may have a bug for the XMPP links. I
will check.
